lib/perl/Genome/Model/Tools/Picard.pm

```diff
@@ -145,7 +145,6 @@ EOS
 # Please put the most recent first.
 my $sw_legacy_java = Genome::Config::get('sw_legacy_java');
 my @PICARD_VERSIONS = (
-    '1.123' => '/gscmnt/sata132/techd/solexa/jwalker/lib/picard-tools-1.123',
```

Comment:
You are correct. After second look on path_for_picard_version, it shows if no version is found in legacy hash, it will try to find one (like 1.123) from /usr/share/java/picard-tools*, which is kind of odd and might need refactoring later but is out of scope of this PR.  

+1